### PR TITLE
Boolean nodes - 1

### PR DESCRIPTION
**Pull request by jinleevv**

### CODE REVIEW

File: blocks/DATA/GENERATION/SIMULATIONS/BOOLEAN/BOOLEAN.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+@flojoy
+def BOOLEAN(status: bool = True) -> Boolean:
+    """The MATRIX node takes two arguments, 'row' and 'col', as input.
+    Based on these inputs, it generates a random matrix where the integers inside the matrix are between 0 and 19.
+
+    Parameters
+    ----------
+    status : True
```
Comment (itsjoeoui) [inline blocks/DATA/GENERATION/SIMULATIONS/BOOLEAN/BOOLEAN.py]: status : bool, default=True

**Review by itsjoeoui — DISMISSED** [review]

File: blocks/DATA/GENERATION/SIMULATIONS/BOOLEAN/BOOLEAN.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from flojoy import flojoy, Boolean
+
+
+@flojoy
+def BOOLEAN(status: bool = True) -> Boolean:
+    """The MATRIX node takes two arguments, 'row' and 'col', as input.
```
Comment (dstrande) [inline blocks/DATA/GENERATION/SIMULATIONS/BOOLEAN/BOOLEAN.py]: Update the docstring

File: blocks/DATA/GENERATION/SIMULATIONS/VECTOR/VECTOR.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+from flojoy import flojoy, Vector
+from numpy import array
+from typing import Literal
+
+
+@flojoy
+def VECTOR(
+    elements: str = "", elements_type: Literal["boolean", "integer"] = "boolean"
```
Comment (dstrande) [inline blocks/DATA/GENERATION/SIMULATIONS/VECTOR/VECTOR.py]: Should this be boolean and numeric (i.e float and integer)

File: blocks/ETL/TRANSFORM/TYPE_CASTING/VECTOR_2_SCALAR/VECTOR_2_SCALAR.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+from numpy import bool_, int_
+from flojoy import flojoy, Vector, Scalar
+
+
+@flojoy
+def VECTOR_2_SCALAR(default: Vector) -> Scalar:
+    """The VECTOR_2_SCALAR node takes a vector and transform it into scalar data type
+
```
Comment (dstrande) [inline blocks/ETL/TRANSFORM/TYPE_CASTING/VECTOR_2_SCALAR/VECTOR_2_SCALAR.py:11]: Add a line saying how the scalar is calculated. Sum or whatever

**Review by dstrande — CHANGES_REQUESTED** [review]

File: blocks/CONTROL_FLOW/LOGICAL_OPERATORS/AND/AND.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+from flojoy import flojoy, Boolean
+
+
+@flojoy
+def AND(default: Boolean, a: Boolean) -> Boolean:
+    """The AND node takes two boolean data type and computs logical AND operation on them.
```
Comment (dstrande) [inline blocks/CONTROL_FLOW/LOGICAL_OPERATORS/AND/AND.py]: Remove the first two works on all blocks and change `node` to `block` for all blocks

**Review by dstrande — CHANGES_REQUESTED** [review]
Hey sorry two more changes:

1. Remove the first couple words in the docstring so its just: `Takes two boolean ...`
2. Change all `node` references to `block` which is what we changed the name to

**Review by dstrande — APPROVED** [review]

**Pull request merged**